xbmc/music/Song.cpp

```diff
@@ -165,8 +165,8 @@ const std::vector<std::string> CSong::GetArtist() const
   //When artist credits have not been populated attempt to build an artist vector from the descrpition string
   //This is a tempory fix, in the longer term other areas should query the song_artist table and populate
   //artist credits. Note that splitting the string may not give the same artists as held in the song_artist table
-  if (songartists.empty() && !strArtistDesc.empty())
-    songartists = StringUtils::Split(strArtistDesc, g_advancedSettings.m_musicItemSeparator);
+  //if (songartists.empty() && !strArtistDesc.empty())
+  //  songartists = StringUtils::Split(strArtistDesc, g_advancedSettings.m_musicItemSeparator);
```

Comment:
More commented out code. I'm not really a fan. I know this from work, usually you forget about it and half a year later someone looks at it and has no clue why it's commented out. :/